addins/Cake.Watch.yml

```diff
@@ -2,7 +2,7 @@ Name: Cake.Watch
 NuGet: Cake.Watch
 Assemblies:
 - "/**/Cake.Watch.dll"
-Repository: https://github.com/cake-addin/cake-watch
+Repository: https://www.nuget.org/packages/Cake.Watch/
```

Comment:
Same with this one.
```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>Cake.Watch</id>
    <version>0.2.2</version>
    <authors>Cake.Watch</authors>
    <owners>Cake.Watch</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <iconUrl>https://cdn.rawgit.com/cake-contrib/graphics/a5cf0f881c390650144b2243ae551d5b9f836196/png/cake-contrib-medium.png</iconUrl>
    <description>Package Description</description>
    <dependencies>
      <group targetFramework=".NETStandard2.0">
        <dependency id="Cake.Core" version="0.30.0" exclude="Build,Analyzers" />
      </group>
    </dependencies>
  </metadata>
</package>
```

Reply:
In this case I think there are 2 problems:
1. The author has omitted the projectURL from their nuspec which caused the AddinDiscoverer to fallback on the nuget url
2. The name of the package (Cake.Watch) does not match the name of the project under the cakecontrib organization (cake-watch). Therefore AddinDiscoverer was unable to detect that this project was transfered.

Reply:
This is similar scenario as Cake.NDepend [discussed previously](https://github.com/cake-build/website/pull/660/files/5e12dfd10aee98ff734737c6c5554cd484b853cb#r295908842).

Reply:
I submitted a [PR](https://github.com/wk-j/cake-watch/pull/13) in this addin repo to add the necessary URL in their nuspec.